src/contexts/LoggedOnUser.jsx

```diff
@@ -0,0 +1,11 @@
+import { createContext, useState } from "react";
+
+export const LoggedOnUserContext = createContext()
+
+export const LoggedOnUserProvider = ({children}) => {
+
+    // logged on user is hard coded for now
+    const [loggedOnUser, setLoggedOnUser] = useState("tickle122")
+
+    return <LoggedOnUserContext.Provider value={{loggedOnUser, setLoggedOnUser}}>{children}</LoggedOnUserContext.Provider>
+}
```

Comment:
Great use case for context! Something used throughout the app in many places and would need a lot of prop drilling otherwise